python/rpdk/java/templates/init/guided_aws/StubCreateHandler.java

```diff
@@ -0,0 +1,138 @@
+            new ReadHandler().handleRequest(proxy, request, callbackContext, logger);
+            throw new CfnAlreadyExistsException(ResourceModel.TYPE_NAME, Objects.toString(model.getPrimaryIdentifier()));
+        } catch (CfnNotFoundException e) {
+            logger.log(model.getPrimaryIdentifier() + " does not exist; creating the resource.");
+            return ProgressEvent.progress(model, callbackContext);
+        }
+    }
+
+    /**
+     * Implement client invocation of the create request through the proxyClient, which is already initialised with
+     * caller credentials, correct region and retry settings
+     * @param awsRequest the aws service request to create a resource
+     * @param proxyClient the aws service client to make the call
+     * @return awsResponse create resource response
+     */
+    private AwsResponse createResource(
+        final AwsRequest awsRequest,
+        final ProxyClient<SdkClient> proxyClient) {
+        AwsResponse awsResponse = null;
+        try {
+            // TODO: add custom create resource logic
+            // e.g. https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs/blob/master/aws-logs-loggroup/src/main/java/software/amazon/logs/loggroup/CreateHandler.java#L37-L43
+
+        // Proxy framework handles the majority of standardized aws exceptions
+        // Example of error handling in case of a non-standardized exception
+        } catch (final AwsServiceException e) {
+            throw new CfnGeneralServiceException(ResourceModel.TYPE_NAME, e);
+        }
+
+        logger.log(String.format("%s successfully created.", ResourceModel.TYPE_NAME));
+        return awsResponse;
+    }
+
+    /**
+     * If your service does not provide strong consistency, you will need to account for eventual consistency issues
+     * ensure you stabilize your Create request so that a subsequent Read will successfully describe the resource state that was provisioned
+     * @param awsRequest the aws service request to create a resource
+     * @param awsResponse the aws service response to create a resource
+     * @param proxyClient the aws service client to make the call
+     * @param model resource model
+     * @param callbackContext callback context
+     * @return boolean state of stabilized or not
+     */
+    private boolean stabilizedOnCreate(
+        final AwsRequest awsRequest,
+        final AwsResponse awsResponse,
+        final ProxyClient<SdkClient> proxyClient,
+        final ResourceModel model,
+        final CallbackContext callbackContext) {
+
+        // TODO: add custom stabilization logic
+
+        // hint: for some resources just describing current status would be enough
+        // e.g. https://github.com/aws-cloudformation/aws-cloudformation-resource-providers-logs/blob/master/aws-logs-loggroup/src/main/java/software/amazon/logs/loggroup/ReadHandler.java#L40-L46
```

Comment:
why is this a link to the read handler? actually, it looks like log group does not stabilize at all.